package.json

```diff
@@ -37,7 +37,7 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.1",
     "nyc": "^11.1.0",
-    "prettier": "^1.7.2",
+    "prettier": "~1.8.0",
```

Comment:
did you change it intentionally to the `~` range?

Reply:
Yes. In order to have this type of issue where a minot release create an issue for a small change in style.
`patch`shouldn't create any problem. When there is a `minor` or `major` Greenkeeper will open a PR so we have time to handle the style change if necessary.